app/services/provider_interface/send_organisation_permissions_emails.rb

```diff
@@ -0,0 +1,29 @@
+module ProviderInterface
+  class SendOrganisationPermissionsEmails
+    def initialize(provider_user:, permissions:, set_up: false)
+      @provider_user = provider_user
+      @permissions = permissions
+      @partner_organisation = @permissions.partner_organisation(provider_user)
```

Comment:
Just to check, if a user belongs to both orgs in a relationship, we email the users on the training provider side. Is that confirmed to be the behaviour we want (rather than emailing both sides for example)

Reply:
@benswannack I think this code is too naive. If a user belongs to both orgs we need to understand which side of the relationship was changed in order to notify the users on the partner org. Will fix.

Reply:
I think this should email both sides omitting the user who made the changes. I've asked for clarification on this though.

Reply:
It's actually a bit more complicated than this so I've added a fix in a separate commit.
If users belong to both organisations, whether they are the user making the change or are just eligible recipients, they should only receive one email. We de-duplicate based on the alphabetical order of the two orgs.
Changing organisation permissions is done in the context of the training/ratifying provider so use this as the org making the change.
See https://github.com/DFE-Digital/apply-for-teacher-training/pull/5235/commits/07b1b0a3903d892eff3bc03985e03eb3a5c44318